seed/static/seed/js/controllers/inventory_list_controller.js

```diff
@@ -503,6 +504,7 @@ angular.module('BE.seed.controller.inventory_list', [])
       }, {
         name: 'notes_count',
         displayName: '',
+        headerCellTemplate: '<div role="columnheader" ng-class="{ \'sortable\': sortable }" ui-grid-one-bind-aria-labelledby-grid="col.uid + \'-header-text \' + col.uid + \'-sortdir-text\'" aria-sort="{{col.sort.direction == asc ? \'ascending\' : ( col.sort.direction == desc ? \'descending\' : \'none\')}}"><div role="button" tabindex="0" ng-keydown="handleKeyDown($event)" class="ui-grid-cell-contents ui-grid-header-cell-primary-focus" col-index="renderIndex"><span ui-grid-one-bind-id-grid="col.uid + \'-sortdir-text\'" aria-label="{{getSortDirectionAriaLabel()}}"><i ng-class="{ \'ui-grid-icon-up-dir\': col.sort.direction == asc, \'ui-grid-icon-down-dir\': col.sort.direction == desc, \'ui-grid-icon-up-dir translucent\': !col.sort.direction }" title="{{isSortPriorityVisible() ? i18n.headerCell.priority + \' \' + ( col.sort.priority + 1 ) : null}}" aria-hidden="true"></i><sub ui-grid-visible="isSortPriorityVisible()" class="ui-grid-sort-priority-number">{{col.sort.priority + 1}}</sub></span></div></div>',
```

Comment:
This is dense... 👀 

At the surface, this looks reasonable, but I didn't drill too far down into the custom(?) UI-grid decorators.

Reply:
Haha yeah, but it's not nearly as complicated as it looks.  By default it uses `ui-grid/uiGridHeaderCell`, so I took the default, stripped out the pieces I didn't want, and tweaked a few things for the translucent icon.  Here's a diff that's a little easier to read:
![image](https://user-images.githubusercontent.com/411466/111008074-8a3e6f00-834d-11eb-840b-8aa8db250ac3.png)